Source/Core/InputCommon/ControllerInterface/Win32/Win32.cpp

```diff
@@ -19,20 +19,27 @@
 
 constexpr UINT WM_DOLPHIN_STOP = WM_USER;
 
-static Common::Event s_done_populating;
-static std::atomic<HWND> s_hwnd;
-static HWND s_message_window;
+static Common::Event s_received_device_change_event;
+static HWND s_hwnd;            // Dolphin's render window
```

Comment:
Each of these comments should be *above* their respective lines to avoid clang-format reformatting these lines when new comments are added. e.g.

```cpp
// Dolphin's render window
static HWND s_hwnd;
```